app/models/user.rb

```diff
@@ -100,6 +100,7 @@ def self.name_format
 
   after_create :welcome_mail
   after_create :set_default_widgets
+  after_initialize :add_select_all_taxonomies
```

Comment:
This seems to work for new users but I think we should do similar check for existing ones, e.g. when I update some user through API and select subset of organizations. I can't do this through WebUI because of disabled fields but API/hammer could still do it.